cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -152,9 +151,9 @@ public KafkaBrokerConfigurationBuilder withCruiseControl(String clusterName, Cru
     public KafkaBrokerConfigurationBuilder withStrimziMetricsReporter(MetricsModel model)   {
         if (model instanceof StrimziMetricsReporterModel reporterModel) {
             printSectionHeader("Strimzi Metrics Reporter configuration");
-            writer.println("prometheus.metrics.reporter.listener.enable=true");
-            writer.println("prometheus.metrics.reporter.listener=http://:" + StrimziMetricsReporterModel.METRICS_PORT);
-            writer.println("prometheus.metrics.reporter.allowlist=" + reporterModel.getAllowList());
+            writer.println(StrimziMetricsReporterConfig.LISTENER_ENABLE + "=true");
```

Comment:
Am I right in understanding that these changes are here because we added the StrimziMetricsReporterConfig class in this PR?